dogs-initial/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>[%d{yyyy-MM-dd' 'HH:mm:ss.SSS}] [%25.25class] [%thread] [%line] [%-5level] %m%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <File>/tmp/var/mylog.log</File>
+        <append>true</append>
+        <encoder>
+            <pattern>[%d{yyyy-MM-dd' 'HH:mm:ss.SSS}] [%C] [%t] [%L] [%-5p] %m%n</pattern>
+        </encoder>
+
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+
+            <!-- rollover daily -->
+            <fileNamePattern>/tmp/var/mylog.%d{yyyy-MM-dd HH}.%i.txt</fileNamePattern>
+            <maxFileSize>10MB</maxFileSize>
+
+            <!-- keep 30 days' worth of history -->
+            <maxHistory>30</maxHistory>
+            <totalSizeCap>3GB</totalSizeCap>
+        </rollingPolicy>
+    </appender>
+
+    <appender name="TOMCAT" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <File>/tmp/var/myTomcatLog.log</File>
+        <append>true</append>
+        <encoder>
+            <pattern>[%d{yyyy-MM-dd' 'HH:mm:ss.sss}] [%C] [%t] [%L] [%-5p] %m%n</pattern>
+        </encoder>
+
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+
+            <!-- rollover daily -->
+            <fileNamePattern>/tmp/var/myTomcatLog.%d{yyyy-MM-dd HH}.%i.txt</fileNamePattern>
+            <maxFileSize>10MB</maxFileSize>
+
+            <!-- keep 30 days' worth of history -->
+            <maxHistory>30</maxHistory>
+            <totalSizeCap>3GB</totalSizeCap>
+        </rollingPolicy>
+    </appender>
+
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT"/>
+        <appender-ref ref="FILE"/>
+    </root>
+
+    <logger name="org.apache.catalina" level="OFF">
+        <appender-ref ref="TOMCAT" />
+        <appender-ref ref="STDOUT" />
+    </logger>
+
+    <logger name="org.apache.coyote" level="OFF">
+        <appender-ref ref="TOMCAT" />
+        <appender-ref ref="STDOUT" />
+    </logger>
```

Comment:
To keep Tomcat logs separate from the application logs in the console, take out the "STDOUT" `appender-ref` tag here.